outputs.tf

```diff
@@ -64,3 +64,10 @@ output "verified_domain_name" {
   description = "List of Custom Domain Name"
 }
 
+output "apphub_service_uri" {
+  value       = {
+    service_uri = "//run.googleapis.com/${google_cloud_run_service.main.id}"
+    service_id  = google_cloud_run_service.main.id
```

Comment:
Similar to other comment for `service_id`!

Reply:
Thanks for pointing out this